doc/how_to_guides/stomp_planner/stomp_planner.rst

```diff
@@ -0,0 +1,109 @@
+
+- *max_rollouts*: the combined maximum number of old and new rollouts that are considered for optimization at any iteration.
+
+- *exponentiated_cost_sensitivity:* a factor for tweaking the exponentated costs used for probability calculation. Higher factors make STOMP optimize and converge quicker with the drawback of reduced robustness and less repeatable results.
+
+- *control_cost_weight*: this is the percentage of the trajectory accelerations cost to be applied in the total cost calculation.
+
+- *delta_t*: assumed time change between consecutive points.
+
+- *path_marker_topic*: Name of the topic RViZ subscribes to for path visualization.
+
+Choosing parameters for STOMP requires lesser intuition than CHOMP. One can have the default parameters for STOMP and this works well in most environments. However you could increase the number of timesteps or number of rollouts for STOMP to perform well under more complicated environments.
+
+
+Difference between plans obtained by STOMP, CHOMP and OMPL
+----------------------------------------------------------
+
+In this section a distinction is made between paths obtained from STOMP, CHOMP and OMPL. STOMP.
+Some of the MoveIt planners tend to produce jerky trajectories and may introduce unnecessary robot movements.
+A post processing smoothing step is usually needed.
+In contrast as STOMP tends to produce smooth well behaved motion plans in a short time, there is no need for a post processing smoothing step as required by some other motion planners.
+
+CHOMP is an optimizing planner that optimizes a given initial naive trajectory based on convarient and functional gradient approaches.
+
+OMPL is an open source library for sampling-based motion planning algorithms which primarily rely on random sampling and graph search.
+Sampling-based algorithms are probabilistically complete: a solution will be eventually found if one exists, however non-existence of a solution cannot be reported.
+These algorithms are efficient and usually find a solution quickly.
+Below is a short overview of planner qualities comparing these different approaches:
+
+- **Local Minima Handling**: STOMP can avoid local minima due to its stochastic nature. CHOMP however is prone to and often gets stuck in local minima, thereby failing to find an optimal solution. As per the STOMP and CHOMP papers, STOMP performs better in most cases.
+
+- **Planning Time**: The planning times of STOMP and CHOMP are comparable, even though CHOMP requires more iterations to achieve success than STOMP. This is mainly because each iteration of STOMP requires multiple trajectory cost evaluations, but can make larger steps in a more stable fashion than the CHOMP gradient update rule. OMPL algorithms tend to be quicker in general and will even take comparably less time in difficult planning scenarios while potentially compromising on qualities like path length or smoothness.
```

Comment:
You could mention here that in contrast to CHOMP and STOMP most OMPL algorithms aren't optimizing the solution